pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -520,39 +519,44 @@ func (r *ReconcileNetworkAddonsConfig) stopTrackingObjects() {
 	r.statusManager.SetFromPods()
 }
 
-func updateObjectsLabels(crLables map[string]string, objs []*unstructured.Unstructured) error {
+func updateObjectsLabels(crLabels map[string]string, objs []*unstructured.Unstructured) error {
 	var err error
 	for _, obj := range objs {
 		labels := obj.GetLabels()
 		if labels == nil {
 			labels = map[string]string{}
 		}
-		// Label objects with version of the operator they were created by
-		labels[cnaov1.SchemeGroupVersion.Group+"/version"] = operatorVersionLabel
-		labels[names.PROMETHEUS_LABEL_KEY] = names.PROMETHEUS_LABEL_VALUE
-		labels[names.MANAGED_BY_LABEL_KEY] = names.MANAGED_BY_LABEL_DEFAULT_VALUE
+		if !isOperatorNamespace(obj) {
```

Comment:
Reduce identitation we don't need the else here.
```suggestion
		if isOperatorNamespace(obj) {
		    delete(labels, names.KUBEMACPOOL_CONTROL_PLANE_KEY)    
		}
		....
```

Reply:
We need to do lines 530 - 545 for non operator namespace, 
else we need to do only line 547, we can't do out of the box what you suggested, unless you mean something else?